Updates for the `preview-3` release (#359)
dotnet/docs-aspire: docs/fundamentals/setup-tooling.md

# [Visual Studio](#tab/visual-studio)

> Starting with Visual Studio 2022 17.10 Preview 1 includes the **.NET Aspire preview 3** workload by default. To verify that you have the .NET Aspire workload installed, run the following command:

Reviewer: ```suggestion
Visual Studio 2022 17.10 Preview 1 or higher includes the **.NET Aspire preview 3** workload by default. To verify that you have the .NET Aspire workload installed, run the following command:
```
